whist/server/api/ranking/leaderboard.py

```diff
@@ -9,14 +9,17 @@
 
 
 @router.get('/{order}', response_model=list[Player])
```

Comment:
What do you think about the order being in the path? I think it also should be a parameter.